frontend/src/App.jsx

```
@@ -7,6 +7,8 @@ import About from './User/pages/About/About'
 import Projects from './User/pages/Projects/Projects'
 import Mentors from './User/pages/Mentors/Mentors'
 import Auth from './User/pages/Mentors/Login'
+import Leaderboard  from './User/pages/Leaderboard/Leaderboard';
+import Error  from './User/pages/Error/Error';
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)